mjpc/utilities.cc

```diff
@@ -585,7 +585,8 @@ mjtNum Ground(const mjModel* model, const mjData* data, const mjtNum pos[3]) {
                        bodyexclude, &geomid);
 
   if (dist < 0) {  // SHOULD NOT OCCUR
-    mju_error("no group 0 geom detected by raycast");
+    return 0.0;
+    // mju_error("no group 0 geom detected by raycast");
```

Comment:
ummm, why?
perhaps the humanoid geom groups need to change?

Reply:
This method oftens errors for humanoid. Maybe I didn't set all of the geom `group` specifications correctly?